src/validators/literal.rs

```diff
@@ -71,7 +70,7 @@ impl<T: Debug> LiteralLookup<T> {
                     .map_err(|_| py_schema_error_type!("error extracting str {:?}", k))?;
                 expected_str.insert(str.to_string(), id);
             } else if expected_py_dict.set_item(&k, id).is_err() {
-                expected_py_list.append((&k, id))?;
+                expected_py_values.push((k.as_unbound().clone_ref(py), id));
```

Comment:
Just trying to learn more about rust syntax here - could you explain this `as_unbound.clone_ref(py)`? Thanks!

Reply:
As discussed offline - is PyO3 code to change from `Bound<'py, PyAny>` to `Py<PyAny>`.

Reply:
Nice. I'll approve!